### PR TITLE
Update "navigating mosviz UI" docs

### PR DESCRIPTION
**Pull request by javerbukh**

### CODE REVIEW

File: docs/mosviz/navigation.rst [hunk 1]
```diff
@@ -2,7 +2,17 @@
...
 
-Overview here...
+The Mosviz User Interface contains an image viewer in the top left section and
+a 2D spectrum viewer over a specviz viewer in the top right section. The bottom
+section houses the :ref:`table viewer<table-viewer>`.
+The image viewer contains all of the same functionality as an image viewer
+in Cubeviz, with the addition that there is a :ref:`slit overlay<slit-overlay>` that uses data from a
+spectrum 2d file that is linked to the image file to determine where to plot the slit.
+The 2D spectrum viewer and the specviz viewer have their x axes’ locked, meaning that
+when you Pan/Zoom in one viewer, the exact same action happens in the other.
```
Comment (rosteen) [inline docs/mosviz/navigation.rst:12]:
```suggestion
The 2D spectrum viewer and the 1D spectrum viewer viewer have their x axes locked to one another, meaning that
when you pan or zoom the x axis in one viewer, the same action happens in the other.
```

Minor wording suggestions

**Review by rosteen — CHANGES_REQUESTED** [review]
Personally I'm not a fan of calling the 1D spectrum viewers "specviz viewers", I would probably mirror what you said for Cubeviz by calling it the 1D spectrum viewer and noting that it brings along the Specviz functionality with it.

I also think that it might be useful to have an image of the UI in this section, so the reader can see what the docs are talking about here.

File: docs/mosviz/navigation.rst (hunk 1 above)
Comment (javerbukh) [reply docs/mosviz/navigation.rst]: Good suggestions! Yeah I had it as "1D spectrum viewer" originally but I wasn't sure if we wanted to stick with that terminology or start to shift towards "specviz viewer". I agree though that it's better to stay consistent until we all agree on something else.

**Review by rosteen — APPROVED** [review]
Looks good to me!

**Review by PatrickOgle — APPROVED** [review]
Approved, echoing suggestion to use "1D spectrum viewer", rather than "specviz viewer".

**Review by rosteen — APPROVED** [review]
I guess my previous approval got dropped when updates were made. Approving again, looks good.

**Pull request merged**